docs/general/ad-filtering/create-own-filters.md

```diff
@@ -267,6 +270,26 @@ In the following examples it is implied that requests are sent from `http://exam
 >
 > Safari does not support the simultaneous use of allowed and disallowed domains, so rules like `||baddomain.com^$domain=example.org|~foo.example.org` will not work in AdGuard for Safari.
 
+> Starting with CoreLibs version 1.12, the `$domain` modifier can be alternatively spelled as `$from`.
```

Comment:
such notes should have a subtitle `Compatibility with different versions of AdGuard`